build/Dockerfile

```diff
@@ -1,7 +1,7 @@
 
 #----------------------- base -----------------------
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9-1108.1705420507 AS base
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9-1108.1706795067 AS base
```

Comment:
@rexwhite any reason to use a specific tag like `8.9-1108.1706795067` if you drop this and say `FROM registry.access.redhat.com/ubi8/ubi-minimal` it should pull the latest all the time isnt it?

Reply:
Yes!  Doing that, or basing images on ":latest" is actually an anti-pattern.  The problem is that anytime you rebuild the image you run the risk of the parent image version changing unexpectedly.  This makes builds non-deterministic, for starters. and changes in parent image version need to run through QE _anyway_, which we can't really automate for Remediations...